### PR TITLE
Add details about copyright notices and donated repos

### PR DESCRIPTION
**Pull request by swinslow**
Ref: https://groups.google.com/d/topic/kubernetes-sig-architecture/TjHLgJcDF-I/discussion

Ref: #3427 

/cc @dims @justaugustus @nikhita 
licensing subproject members

/assign @nikhita
/hold

This is a replacement for #3427, modifying the language and adding some details about copyright transfers and copyright notices.

Agree with comment at https://github.com/kubernetes/community/pull/3427#issuecomment-475847195 that sig-release might be a better place to aggregate more information about copyright notices, etc. over the longer term. I'm thinking that the added details here will be useful for now as we build out the more detailed guidance.

### CODE REVIEW

File: github-management/kubernetes-repositories.md [hunk 1]
```diff
@@ -98,6 +98,13 @@ the developers who could not be reached
      [@caniszczyk](https://github.com/caniszczyk) for review of third party deps
    * Boilerplate text across all files should attribute copyright as follows:
      `"Copyright <Project Authors>"` if no CLA was in place prior to donation
+   * Additions of [the standard Kubernetes header](https://github.com/kubernetes/kubernetes/blob/master/hack/boilerplate/boilerplate.go.txt)
```
Comment (nikhita) [inline github-management/kubernetes-repositories.md]: The line above says `Copyright <Project Authors>`, can we modify that to point to the standard Kubernetes header as well?

File: github-management/kubernetes-repositories.md [hunk 2]
```diff
@@ -220,6 +227,20 @@ associated projects. Many things seem vague. Help!**
...
+**When I donate my project, am I transferring my copyrights?**
+
+No. All contributors retain ownership of their copyrights in the code they donate.
+Instead, they are granting a license to the project (that's the 'L' in 'CLA').
+
+For consistency and efficiency in complying with notice requirements, code that is
+donated to a Kubernetes repo should use [the standard header](https://github.com/kubernetes/kubernetes/blob/master/hack/boilerplate/boilerplate.go.txt)
+referencing "The Kubernetes Authors". That doesn't mean you are transferring your
+copyright. Instead, it's a general reference to the fact that the copyrights remain
+owned by the authors of Kubernetes.
```
Comment (nikhita) [inline github-management/kubernetes-repositories.md]: I'm a little confused :see_no_evil: Just for my understanding:

> That doesn't mean you are transferring your copyright. Instead, it's a general reference to the fact that the copyrights remain owned by the authors of Kubernetes.

Does "remain owned" mean that the ownership of the copyright is transferred from the original authors/original company to the authors of Kubernetes?

Comment (swinslow) [reply github-management/kubernetes-repositories.md]: Hi @nikhita, no worries, if you are finding it confusing then I should reword it to make it clearer =)

The idea is that ownership for contributed code does not change. The owners of the copyrights might be individuals, or their employers, but in any case whoever owns the copyright before contribution still owns it afterwards. It’s just that a license is granted under the CLA. 

Let me stare at the language a bit more and try to clarify it better.

Comment (nikhita) [reply github-management/kubernetes-repositories.md]: I'm ok with merge and iterate-if-needed, if @dims and @justaugustus are good with this.

**Comment by justaugustus** [comment]
Yes, let's iterate.
/lgtm
/approve

**Comment by nikhita** [comment]
@dims (licensing subproject owner) and @spiffxp (github-management subproject owner) - can you add explicit lgtms if you are good with this? Thanks!

**Comment by nikhita** [comment]
/assign @cblecker 
github-management owner

File: github-management/kubernetes-repositories.md (hunk 1 above)
Comment (cblecker) [inline github-management/kubernetes-repositories.md]:
```suggestion
   * Additions of [the standard Kubernetes header](https://git.k8s.io/kubernetes/hack/boilerplate/boilerplate.go.txt)
```

File: github-management/kubernetes-repositories.md [hunk 3]
```diff
@@ -220,6 +227,20 @@ associated projects. Many things seem vague. Help!**
 True, we need to improve these things. For now, do the best you can to conform
 to the spirit of the proposal (e.g. post the code of conduct, etc)
 
+**When I donate my project, am I transferring my copyrights?**
+
+No. All contributors retain ownership of their copyrights in the code they donate.
+Instead, they are granting a license to the project (that's the 'L' in 'CLA').
+
+For consistency and efficiency in complying with notice requirements, code that is
+donated to a Kubernetes repo should use [the standard header](https://github.com/kubernetes/kubernetes/blob/master/hack/boilerplate/boilerplate.go.txt)
```
Comment (cblecker) [inline github-management/kubernetes-repositories.md]:
```suggestion
donated to a Kubernetes repo should use [the standard header](https://git.k8s.io/kubernetes/hack/boilerplate/boilerplate.go.txt)
```

**Comment by cblecker** [comment]
This looks fine to me, if we can make the two link shortener changes to comply with our style guidelines.

File: github-management/kubernetes-repositories.md (hunk 1 above)
Comment (swinslow) [reply github-management/kubernetes-repositories.md]: Added in 3edd86c -- thanks!

File: github-management/kubernetes-repositories.md (hunk 3 above)
Comment (swinslow) [reply github-management/kubernetes-repositories.md]: Also added in 3edd86c -- thanks!

**Comment by cblecker** [comment]
/lgtm
/approve

Thanks @swinslow!
